msgcntr/messageforums-hbm/src/java/org/sakaiproject/component/app/messageforums/dao/hibernate/DraftRecipientImpl.java

```diff
@@ -0,0 +1,73 @@
+package org.sakaiproject.component.app.messageforums.dao.hibernate;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.sakaiproject.api.app.messageforums.DraftRecipient;
+import org.sakaiproject.api.app.messageforums.MembershipItem;
+
+@Getter @Setter
+public class DraftRecipientImpl implements DraftRecipient
```

Comment:
POJO's have little need for an interface and an impl move this to the interface and make it class

Reply:
I was just following the pattern used in the rest of the tool, but I agree so I'll change it.